eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedView.scala

```diff
@@ -133,6 +136,12 @@ trait EventsourcedView extends Actor with Stash {
     settings.replayBatchSize
 
   /**
+   * Maximum number of replay attempts before finally stopping the actor itself.
+   */
+  def maxReplayAttempts: Int =
```

Comment:
This method is only overridden in tests. Why do we need this method instead of a special test config?